torchaudio/functional.py

```diff
@@ -461,6 +464,12 @@ def create_fb_matrix(
     down_slopes = (-1.0 * slopes[:, :-2]) / f_diff[:-1]  # (n_freqs, n_mels)
     up_slopes = slopes[:, 2:] / f_diff[1:]  # (n_freqs, n_mels)
     fb = torch.max(zero, torch.min(down_slopes, up_slopes))
+
+    if str(norm) == "slaney":
```

Comment:
Why the cast? You could raise an exception if the type doesn't match if it's not None.

Reply:
Without the cast:
```
======================================================================
ERROR: test_create_fb_matrix (__main__.TestFunctionalCPU)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test/test_torchscript_consistency.py", line 106, in test_create_fb_matrix
    self._assert_consistency(func, dummy)
  File "test/test_torchscript_consistency.py", line 38, in _assert_consistency
    return _assert_functional_consistency(func, tensor, self.device, shape_only=shape_only)
  File "test/test_torchscript_consistency.py", line 14, in _assert_functional_consistency
    ts_func = torch.jit.script(func)
  File "/Users/vincentqb/anaconda3/envs/audio-built/lib/python3.7/site-packages/torch/jit/__init__.py", line 1281, in script
    fn = torch._C._jit_script_compile(qualified_name, ast, _rcb, get_default_args(obj))
  File "/Users/vincentqb/anaconda3/envs/audio-built/lib/python3.7/site-packages/torch/jit/_recursive.py", line 555, in try_compile_fn
    return torch.jit.script(fn, _rcb=rcb)
  File "/Users/vincentqb/anaconda3/envs/audio-built/lib/python3.7/site-packages/torch/jit/__init__.py", line 1281, in script
    fn = torch._C._jit_script_compile(qualified_name, ast, _rcb, get_default_args(obj))
RuntimeError: 
Could not cast value of type Optional[str] to bool:
'create_fb_matrix' is being compiled since it was called from 'func'
  File "test/test_torchscript_consistency.py", line 103
            sample_rate = 16000
            norm = None
            return F.create_fb_matrix(n_stft, f_min, f_max, n_mels, sample_rate, norm)
                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE


----------------------------------------------------------------------
Ran 1 test in 0.032s

FAILED (errors=1)
```